config/license_finder.yml

```diff
@@ -0,0 +1,7 @@
+---
+project_name: Porta
+dependencies_file_dir: './doc/license'
+decisions_file: './doc/licenses/dependency_decisions.yml'
```

Comment:
is it intentional to have `./doc/license` vs `./doc/licenses`?

Reply:
No, it's a typo